start.sh

```diff
@@ -1,10 +1,48 @@
 #!/bin/bash
 
-# start peacock
-cd Peacock
+# Get the directory of the script
+SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+pushd $SCRIPT_DIR
 
-PORT=3000 ../node/bin/node ./chunk0.js &
+# Grab Peacock if needed
+if [ ! -f "./Peacock/chunk0.js" ]; then
+    LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/thepeacockproject/peacock/releases/latest | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
 
-# start game command
-../run
+    wget -O peacock.zip "https://github.com/thepeacockproject/Peacock/releases/download/${LATEST_RELEASE}/Peacock-${LATEST_RELEASE}.zip"
 
+    unzip peacock.zip
+    rm peacock.zip
+
+    mv "Peacock-${LATEST_RELEASE}" Peacock
+fi
+
+# Copy files
+STEAM_DIR=~/.local/share/Steam
+
+STEAM_PATHS=$(cat "${STEAM_DIR}/steamapps/libraryfolders.vdf" | grep -oP '"path"\s+"\K[^"]+')
+
+for i in $STEAM_PATHS; do
+    if [ -d "${i}/steamapps/common/HITMAN 3" ]; then
+        # Hitman installed here
+        cp Peacock/PeacockPatcher.exe "${i}/steamapps/common/HITMAN 3/"
+        cp WineLaunch.bat "${i}/steamapps/common/HITMAN 3/"
+    fi
+done
+
+# Install node if needed
+if ! command -v node &> /dev/null && [ ! -d "./node" ]; then
+    node_version="18.15.0"
```

Comment:
This should ideally read from the `.nvmrc` file included with the Peacock distribution - it only contains the version of Node that Peacock expects to be present in the environment, so it will ensure that it's future proof (Peacock v7 which is in development now uses Node v20!)

Reply:
Ah good call, I'll add this in tomorrow 😄

Reply:
Pushed, will test on my deck in a little while

Reply:
Tested, this works now
![image](https://github.com/thepeacockproject/linux-steam-setup/assets/2872862/5787d87e-8e2b-44a6-acee-655b29a8be36)